kernel-version.mk

```diff
@@ -14,7 +14,7 @@
 KERNEL_COMPILER=gcc
 PLATFORM?=generic
 
-PLATFORMS_amd64=generic rt
+PLATFORMS_amd64=generic rt evaluation
 PLATFORMS_arm64=generic nvidia nvidia-jp5 nvidia-jp6 imx8mp_pollux imx8mp_epc_r3720 imx8mq_evk
```

Comment:
If we (later) add this to arm64 will we end up with one evaulation platform (matching the generic), or potentially one evaluation for each of the current arm64 platforms?